#! /bin/sh /usr/share/dpatch/dpatch-run

## common-vtx-dir patch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use common vtx directory with vdr-plugin-osdteletext.

@DPATCH@
--- vdrdevel-plugin-osdteletext/debian/dirs
+++ vdrdevel-plugin-osdteletext/debian/dirs
@@ -1,1 +1,1 @@
-var/cache/vdrdevel/vtx
+var/cache/vdr/vtx
